### PR TITLE
Privacy policy

### PR DESCRIPTION
**Pull request by 409H**
[CH7734]

Added the MyCrypto privacy policy into landing, maintained with a simple markdown file

Related: https://github.com/MyCryptoHQ/about.mycrypto.com/pull/81

### CODE REVIEW

**Review by vilsbole — COMMENTED** [review]
Code lgtm;
But breaks in staging.
- The link path `/privacy` substitutes the url so removes the pr path eg. `pr/39`
- Since gatsby is all about SSR, it doesn't offer built-in HashRouter. 
Investigating.

File: package.json [hunk 1]
```diff
@@ -24,7 +24,10 @@
     "gatsby-plugin-sitemap": "^2.4.7",
     "gatsby-plugin-styled-components": "3.3.0",
     "gatsby-plugin-typescript": "2.1.7",
+    "gatsby-remark-autolink-headers": "^2.10.0",
+    "gatsby-remark-external-links": "^0.0.4",
     "gatsby-source-filesystem": "2.1.22",
+    "gatsby-transformer-remark": "^2.15.0",
```
Comment (GuillaumeRx) [inline package.json:30]: Since all other deps are fixed versions please fix the version

File: src/components/Link.tsx [hunk 2]
```diff
@@ -1,7 +1,11 @@
 import React from 'react';
+import { Link as GatsbyLink } from 'gatsby';
 import { LinkProps, Link as ExtLink } from 'rebass';
 
 const Link = (props: LinkProps) => {
+  if (props.to) {
+    return <GatsbyLink {...props} />;
```
Comment (GuillaumeRx) [inline src/components/Link.tsx:7]: Please fix link style, this shouldn't be underlined

**Review by GuillaumeRx — CHANGES_REQUESTED** [review]

**Comment by GuillaumeRx** [comment]
Also a TS error to fix

File: package.json (hunk 1 above)
Comment (vilsbole) [reply package.json:30]: Done

File: src/components/Link.tsx (hunk 2 above)
Comment (vilsbole) [reply src/components/Link.tsx:7]: Done

**Review by GuillaumeRx — APPROVED** [review]
lgtm

**Review by GuillaumeRx — APPROVED** [review]

File: src/assets/content/privacy-policy.md [hunk 3]
```diff
@@ -0,0 +1,251 @@
+---
+slug: '/privacy-policy'
+date: '2021-01-22'
+title: 'Privacy Policy'
+---
+
+[MyCrypto](https://mycrypto.com/) values your privacy and security and actively works to collect as little of your personal data and information as possible. Nonetheless, any company providing publicly available services on the internet will inevitably collect information about those who it interacts with or those who utilize its services ("Users") through its normal business operations.
```
Comment (409H) [inline src/assets/content/privacy-policy.md:7]: some changes

**Pull request merged**